src/lib/y2storage/mountable.rb

```diff
@@ -88,6 +88,13 @@ def mountpoint=(path)
 
     alias_method :mount_point=, :mountpoint=
 
+    # Is the mount persistent?
+    # @return [Boolen] true if the mount point is saved to /etc/fstab
+    #   (and will be mounted at boot again), false otherwise
+    def persistent?
+      to_storage_value.mount_point.in_etc_fstab?
```

Comment:
This can raise an exception if there is no mount point. This should be more robust:
```ruby
to_storage_value.has_mount_point && to_storage_value.mount_point.in_etc_fstab?
```